decoration.sk

```diff
@@ -732,30 +739,39 @@ caves and cliffs update part 1:
 	glow item frame [item]¦s = minecraft:glow_item_frame[relatedEntity=glow item frame]
 	
 	(grass|dirt) path [block]¦s = minecraft:dirt_path
-
+	any dirt = any dirt, dirt path
 
 the wild update:
 	minecraft version = 1.19 or newer
 
 	frogspawn¦s = minecraft:frogspawn
-	
+
+	mangrove root¦s = minecraft:mangrove_roots
```

Comment:
already defined at https://github.com/SkriptLang/skript-aliases/blob/e99be898254965b0207992f66a3289ab6744106e/building.sk#L784 so we'll want to remove it there